Resources/public/mapbender.element.digitizer.js

```diff
@@ -1696,7 +1698,8 @@
                     if(!hasErrors) {
                         var formData = form.formData();
                         var uniqueIdKey = schema.dataStore.uniqueId;
-                        var isNew = !dataItem.hasOwnProperty(uniqueIdKey) && !!dataItem[uniqueIdKey];
+                        var isNew = !dataItem.hasOwnProperty(uniqueIdKey) && !dataItem[uniqueIdKey];
```

Comment:
The logic to determine if the entry is a new one wasn´t correct. New items won´t have a uniqueIdKey set